client/src/components/questions/AnswerList.jsx

```diff
@@ -3,14 +3,19 @@ import React, { useState, useEffect } from 'react';
 import { Answer } from './Answer.jsx';
 import styled from 'styled-components';
 
-const StyledAnswerList = styled.div`
+const StyledAnswerContainer = styled.div`
   display: flex;
   justify-content: flex-start;
   max-height: 50vh;
-  overflow: auto;
+  width: max-content;
```

Comment:
Cool! So does this cause the image/content to shrink if there would be overflow? Or does it cut it off?

Reply:
I reconfigured how the structure of the answer list looked so the overflow was moved to the styled component below this one. The "max content" sets the width of the element to the width of the current content, so if there is a longer answer it will stretch the container to hold that sentence up to a certain point. The overflow just makes it so that the list auto scrolls when it gets too long for the container's view.